ntbs-ui-tests/Hooks/DriverSetup.cs

```diff
@@ -75,6 +75,8 @@ private async Task RemoveImportedNotificationAfter()
 
         private async Task CleanUpMigratedNotification()
         {
+            var legacyId = settings.EnvironmentUnderTest == "phe-uat" ? "100894" : "189045";

```

Comment:
100894 was picked arbitrarily

Reply:
I can see that actually these values are used in several places, including in the features. So we would have to use that other config feature that you showed me yesterday.

Reply:
What I was looking at before was using a 'scenario outline' for the one import test, with 'examples' for the two sets of ids, but I had to abandon that method because there were so many steps that needed skipping when on UAT - now we have two different tests, and you can't use one set of examples for two different tests, so it made more sense to move the source of truth for legacyId into the testSettings.